pkg/scheduler/scheduler.go

```diff
@@ -405,12 +406,12 @@ func NewFromConfig(config *factory.Config) *Scheduler {
 }
 
 // Run begins watching and scheduling. It waits for cache to be synced, then starts a goroutine and returns immediately.
```

Comment:
In L414, we changed the behavior from "un-blocking" to "blocking", i.e. we're not starting a goroutine anymore, so the comment should be updated.